catalog-info.yaml

```diff
@@ -37,7 +37,7 @@ spec:
       name: elastic-agent
       description: Buildkite pipeline for the Elastic Agent project
     spec:
-      branch_configuration: "main 7.* 8.* v7.* v8.*"
+      branch_configuration: "main 7.* 8.* 9.* v7.* x8.* v9.*"
```

Comment:
What's the reason for `v[0-9].*`? IIRC, we don't have branches with that naming

```suggestion
      branch_configuration: "main 7.* 8.* 9.* v7.* v8.* v9.*"
```

Reply:
Great question. I was wondering why we even have v7* / v8* so just copied over the same pattern. IIRC @mrodm added those quite some time ago, not sure if there was context he remembers?

Reply:
@pazone do you remember what was the reason for using `v7*`/`v8*` historically for branch filters (in addition to the usual `7.*/8.*` etc) in elastic-agent pipeline configs?

Reply:
@v1v @dliappis IIRC this branch configuration setting is also applied for both branches and tags that Buildkite is going to trigger a build.

If not set, I think it would trigger a build for every tag pushed to the repository (upstream).

For instance , in `package-spec` these are the values 
![image](https://github.com/user-attachments/assets/8047baf2-c133-44e9-81d6-68da17efc3dd)

Reply:
Thanks @mrodm for the clarification. In this case it makes sense to keep this pattern, as this PR does.